src/Console/AbstractConsole.php

```diff
@@ -1,39 +0,0 @@
-<?php
-
-// Copyright (C) 2010-2024, the Friendica project
-// SPDX-FileCopyrightText: 2010-2024 the Friendica project
-//
-// SPDX-License-Identifier: AGPL-3.0-or-later
-
-declare(strict_types=1);
-
-namespace Friendica\Console;
-
-use Asika\SimpleConsole\Console;
-use Friendica\Core\Console as CoreConsole;
-use Friendica\Core\Logger\Capability\LogChannel;
-
-/**
- * Abstract Console class for common settings
- */
-abstract class AbstractConsole extends Console
-{
-	/**
-	 * Overwrite this const in case you want to switch the LogChannel for this console command
-	 *
-	 * @var string
-	 */
-	public const LOG_CHANNEL = LogChannel::CONSOLE;
```

Comment:
Wait, wait @Art4 :D

This will set the log output back to `console` for Daemon and worker calls :)

Reply:
There aren't used at the moment, instead the determination is done in `App->determineLogChannel`